File: 0001-two-sum/0001-two-sum.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(nums, target) {
    let result = []
    for(let i = 0; i<nums.length; ++i){
       for(let j = 0; j<nums.length; ++j){
           if(nums[i] + nums[j] == target && i != j ){
                result.push(i);
                 result.push(j);
           }
       }
   }
    let unique = [...new Set(result)]
    return unique;
    
};

File: 0001-two-sum/0001-two-sum.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        chars_index = dict()
        result = []
        chars_index[nums[0]] = 0
        for i in range(1,len(nums)):
            if target-nums[i] in chars_index:
                num1 = target-nums[i]
                result.append(chars_index[num1])
                result.append(i)
                break
            else:
                chars_index[nums[i]] = i
        return result